homeassistant/components/number/__init__.py

```diff
@@ -218,8 +218,13 @@ async def async_internal_added_to_hass(self) -> None:
     @property
     def capability_attributes(self) -> dict[str, Any]:
         """Return capability attributes."""
-        min_value = self.min_value
-        max_value = self.max_value
```

Comment:
This breaks ability to overload those properties. Not sure if that okey here

Reply:
They are marked as `@final` so they should never be overloaded

Reply:
Ok